protos/telemetry_server/telemetry_server.proto

```diff
@@ -0,0 +1,464 @@
+syntax = "proto3";
+
+package mavsdk.rpc.telemetry_server;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.telemetry_server";
+option java_outer_classname = "TelemetryServerProto";
+
+/*
+ * Allow users to provide vehicle telemetry and state information
+ * (e.g. battery, GPS, RC connection, flight mode etc.) and set telemetry update rates.
+ */
+service TelemetryServerService {
+    // Publish to 'position' updates.
+    rpc PublishPosition(PublishPositionRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'home position' updates.
+    rpc PublishHome(PublishHomeRequest) returns(PublishTelemetryResponse) {}
+    // Publish to armed updates.
+    rpc PublishArmed(PublishArmedRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'Raw GPS' updates.
+    rpc PublishRawGps(PublishRawGpsRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'battery' updates.
+    rpc PublishBattery(PublishBatteryRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'flight mode' updates.
+    rpc PublishFlightMode(PublishFlightModeRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'health' updates.
+    rpc PublishHealth(PublishHealthRequest) returns(PublishTelemetryResponse) {}
+    // Publish to 'status text' updates.
+    rpc PublishStatusText(PublishStatusTextRequest) returns(PublishTelemetryResponse) {}
```

Comment:
Hmm I think that this won't work with the autogen. `PublishStatusText` should have a `PublishStatusTextRequest` and a `PublishStatusTextResponse`. Same for all the others :+1:

Reply:
What will fail? It's compiling and running at the moment

Reply:
Yes, but I know it's a convention I took in the autogen plugin `protoc-gen-mavsdk`. Here it compiles maybe because the response is empty, or maybe because C++ does not require that, but some language bindings will probably need it.

`protoc-gen-mavsdk` leverages `protoc`, and therefore there is no way to change how the proto files look. The convention I took was that a function `functionName()` is defined as `rpc FunctionName(FunctionNameRequest) returns(FunctionNameResponse) {}`.

Maybe it could be done without it, but we would need to try all the language bindings before concluding it :see_no_evil: